src/Models/ProjectModel.php

```diff
@@ -27,13 +27,28 @@ public function getProjects(): array
         return $query->fetchAll();
     }
 
-    public function getById(int $id): ProjectEntity
+    //Not sure about this
+//    public function getTask(): array|false
+//    {
+//        $query = $this->db->prepare(
+//     'SELECT `tasks`.`name` AS "taskname",
+//`tasks`.`estimate`,
+//`users`.`name` as "username",
+// `users`.`avatar` AS "icon"
+// FROM `tasks` INNER JOIN `users`
+// ON `tasks`.`user_id` = `users`.`id`;');
+//        $query->execute([]);
+//        return $query->fetch();
+//    }
```

Comment:
I understand why this might be useful in the future, but if you leave large chunks of commented-out code in pull requests and they get merged, before long you'll have more commented-out code than actual code in your project. So please make a note of it somewhere and remove it!